neqo-bin/src/udp.rs

```diff
@@ -61,7 +60,7 @@ impl Socket {
         let transmit = Transmit {
             destination: d.destination(),
             ecn: EcnCodepoint::from_bits(Into::<u8>::into(d.tos())),
-            contents: d.into_data().into(),
+            contents: Vec::from(d).into(),
```

Comment:
Is this part of the move or another change?

Reply:
`Datagram::into_data` is changed to a `From<Datagram> for Vec<u8>` (see comment below). Instead of calling `into_data`, this line first leverages the new `From` implementation and then, as before, converts it into a `Bytes` (`into` at the end).

None of these methods allocate.